prow/config.yaml

```diff
@@ -310,9 +310,11 @@ postsubmits:
           - image: gcr.io/k8s-testimages/e2e-kubeadm:0.2
             args:
             - "--branch=$(PULL_REFS)"
-            - "--upload=gs://kubernetes-jenkins/logs"
-            - "--git-cache=/root/.cache/git"
             - "--clean"
+            - "--git-cache=/root/.cache/git"
+            - "--json"
+            - "--timeout=140"
```

Comment:
it used to be something like DOCKER_TIMEOUT in dockerized-e2e-runner - https://github.com/kubernetes/test-infra/blob/master/jenkins/dockerized-e2e-runner.sh#L80